.github/workflows/pants.yaml

```diff
@@ -25,9 +25,6 @@ jobs:
     steps:
     - uses: actions/checkout@v3
     - uses: pantsbuild/actions/init-pants@v1
-      # This action bootstraps pants and manages 2-3 GHA caches.
```

Comment:
The first two sentences are still valid, right?

Reply:
ah, yes, just the tag